src/unstable-temp/DownloadImageModal/OsConfiguration.tsx

```diff
@@ -271,7 +271,9 @@ export const OsConfiguration = ({
 			{(!isInitialDefault || !version) && (
 				<Box mb={3}>
 					<DownloadImageLabel>
-						{t('placeholders.select_version')}
+						{hasEsrVersions
+							? t('placeholders.select_version')
+							: t('placeholders.select_balenaos_version')}
```

Comment:
This is more specific, but I'm not sure if this is really necessary. I have no strong opinion on it, but IMHO `select version` is already explicit enough.

Reply:
🤔 It's used when the chosen device does not support ESR in which case the user *must* use OS, but when that happens it's not explicitly stated anywhere that that is what's happening so I thought this might be useful for making the user aware that they are selecting an OS version. I hear you though, perhaps `select_version` is explicit enough on its own and users will assume by default that they are using balenaOS on their device if they are not prompted to select one. I'll make the change :)